### DISCUSSION TITLE
Checkin export

### DISCUSSION THREAD

**Discussion by smokephil** [pretix/pretix#4299] [Q&A] [answered]
Hello,

is it possible to export the check in hints as well? I didn't find an option on the export screen.

Best regards,
Philipp

**Comment by wiffbi** [comment]
You mean the „Check-in text“ that is available on product’s settings tab „Check-in & Validity“? I do not see why not. IMHO makes sense to include it after the „Requires special attention“ column. I will look into this.

**Reply by smokephil** [reply]
I mean the field from an order in the section "internal comment"

**Comment by wiffbi** [comment, marked as answer]
Thanks for the suggestion, we added it in #4300
